offer-manager-app/src/main/resources/offer-template/stylesheet.css

```diff
@@ -136,7 +136,7 @@ h2 {
 
 .signature {
     padding-left: 100px;
-    padding-top: 55px;
+    padding-top: 75px;
```

Comment:
Which is correct here? I think @sven1103 made some adjustments here for the signature in the pdf?

Reply:
Those are these changes, right?

Reply:
ok I checked, 75px is correct